doc/org.eclipse.cdt.doc.user/src/asciidoc/core_build_system.adoc

```diff
@@ -0,0 +1,57 @@
+////
+Copyright (c) 2000-2024 QNX Software Systems and others
+This program and the accompanying materials
+are made available under the terms of the Eclipse Public License 2.0
+which accompanies this distribution, and is available at
+https://www.eclipse.org/legal/epl-2.0/
+
+SPDX-License-Identifier: EPL-2.0
+////
+
+
+// support image rendering and table of contents within GitHub
+ifdef::env-github[]
+:imagesdir: ../../images
+:toc:
+:toc-placement!:
+endif::[]
+
+// enable support for button, menu and keyboard macros
+:experimental:
+
+= Core Build System
+
+The Core Build System is the successor of the Managed Build System.
+
+The classical Makefile projects are based on the Managed Build System.
+Managed Build projects can generate a Makefile based on GUI controlled
+build configurations in the project properties. The user can
+manage the separate build configurations for Run and Debug. There is a
+notion of "active" build configuration, which is the one that gets
+built when the build button is pressed.
+
+The Core Build System is the basis for CMake projects and Core Build
+Makefile projects. The CDT LaunchBar plays a central role and is used
+for building and launching the program. The philosophy is "Build for
+launch". It simplified the flow a lot.
+
+These are the main differences with the old Managed Build System:
+
+* No build configurations. No build file (Makefile or
+  CMakeLists.txt) generation. The user writes it by hand. A
```

Comment:
> The user writes it by hand.

Accented like a benefit. However, for an IDE it may sound like a degradation in a path to a "just colored editor"

Reply:
I think this is answered in comments. Since we cannot invent something better than MBS, the only way is to let user write build file by hand.